README.md

```diff
@@ -25,7 +25,7 @@ pip install reckoner
 
 Via Git
 ```shell
-pip install git+https://github.com/reactiveops/reckoner@master
+pip install git+https://github.com/FairwindsOps/reckoner@master
```

Comment:
In Go I think we have been using lowercase, but @bobby-brennan should verify.

Reply:
Per offline discussion, we're gonna go with lowercase for golang packages, but for copy/paste stuff from READMEs, let's keep it `FairwindsOps`